src/ocm/components/clusterWizard/ClusterPlatformIntegrationHint.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Alert, AlertVariant } from '@patternfly/react-core';
+import { Cluster } from '../../../common';
+import { useClusterSupportedPlatforms } from '../../hooks';
+
+type ClusterPlatformIntegrationHintProps = {
+  cluster: Cluster;
+};
+
+export const ClusterPlatformIntegrationHint: React.FC<ClusterPlatformIntegrationHintProps> = ({
+  cluster,
+}) => {
+  const { isPlatformIntegrationSupported } = useClusterSupportedPlatforms(cluster.id);
+  if (isPlatformIntegrationSupported && cluster?.platform?.type !== 'vsphere') {
```

Comment:
Is this condition correct? Are we displaying the "all your hosts are from vSphere" alert only if the `cluster.platform.type` isn't?

Reply:
Yes, because if the value is already set, it means they already chose the integration with vSphere, then showing the message doesn't make sense.

Reply:
Yes, is like Celia explains